packages/expo-facebook/android/src/main/java/expo/modules/facebook/FacebookModule.kt

```diff
@@ -32,20 +32,17 @@ class FacebookModule(
   context: Context,
   private val moduleRegistryDelegate: ModuleRegistryDelegate = ModuleRegistryDelegate()
 ) : ExportedModule(context), ActivityEventListener {
+  private val ERR_FACEBOOK_MISCONFIGURED = "ERR_FACEBOOK_MISCONFIGURED"
+  private val ERR_FACEBOOK_LOGIN = "ERR_FACEBOOK_LOGIN"
+  private val PUSH_PAYLOAD_KEY = "fb_push_payload"
+  private val PUSH_PAYLOAD_CAMPAIGN_KEY = "campaign"
```

Comment:
I think @lukmccall meant to move these constants outside class definition 😄

Reply:
Yep, exactly 😅